client/src/router/index.js

```diff
@@ -22,6 +17,11 @@ const routes = [
     component: Publication,
     props: true,
   },
+  {
```

Comment:
I'd still really prefer query strings instead of paths, since the filters are unordered and paths require extra config in the nginx routing setup

Reply:
I can see your point. Will take a look.